include/ginkgo/core/base/executor.hpp

```diff
@@ -151,15 +165,16 @@ class ExecutorBase;
  * OMP
  * CUDA(0)
  * HIP(0)
+ * DPC++(0)
```

Comment:
should we use DPC++ or DPCPP?

Reply:
Well that's a good question, so far I use in documentation always `DPC++` I believe, and only during the code I use `DPCPP` since we cannot use the `+` symbol where we want. With the code of this example, it will produce `DPC++(0)` indeed.